wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/tck/WorkspaceDaoTest.java

```diff
@@ -152,14 +167,23 @@ public void shouldThrowNpeWhenGettingWorkspaceByNullNamespace() throws Exception
   }
 
   @Test
-  public void shouldGetWorkspaceByNameAndNamespace() throws Exception {
+  public void shouldGetWorkspaceByConfigNameAndNamespace() throws Exception {
     final WorkspaceImpl workspace = workspaces[0];
 
     assertEquals(
         workspaceDao.get(workspace.getConfig().getName(), workspace.getNamespace()),
         new WorkspaceImpl(workspace));
   }
 
+  @Test
+  public void shouldGetWorkspaceByDevfileNameAndNamespace() throws Exception {
+    final WorkspaceImpl workspace = workspaces[4];
```

Comment:
`4` value actually depends on the value of `COUNT_OF_WORKSPACES`. I think it is better to use `COUNT_OF_WORKSPACES-1` here, or even constant `DEVFILE_WORKSPACE_INDEX` constant